packages/zoe/src/contracts/myFirstDapp.js

```diff
@@ -75,15 +79,26 @@ export const makeContract = harden((zoe, terms) => {
     return 'not an active offer';
   }
 
+  // This is a really simple update protocol, which merely provides a promise
```

Comment:
Why is the order book part of "myFirstDapp"? Are we expecting agoric-sdk to always have an order book? Or is this better suited to a more specialized "myFirstXXXDapp"?

Reply:
It's certainly not the case that all contracts will want to publish an order book. 

I didn't expect that we were planning to build an extensible UI that would support different contracts having different UI elements. The demo we've talked about for alpha wants to have a way to find (and be notified about changes to) the current set of orders. In the absence of a pluggable mechanism for supporting that, I thought we agreed to modify the current code to support the particular exchange we'll have in Alpha.